File: 프로그래머스/unrated/181836. 그림 확대/그림 확대.py
def solution(picture, k):
    answer = []
    for i in range(len(picture)):
        tmp = ""
        for j in range(len(picture[0])):
            tmp += picture[i][j]*k
        for i in range(k):
            answer.append(tmp)

    
    return answer
            
            

File: 프로그래머스/0/181836. 그림 확대/그림 확대.java
import java.util.*;

class Solution {
    public String[] solution(String[] picture, int k) {
        List<String> answer = new ArrayList<>();
        for (String s : picture) {
            StringBuilder tmp = new StringBuilder();
            for (int j = 0; j < picture[0].length(); j++) {
                tmp.append(String.valueOf(s.charAt(j)).repeat(k));
            }
            for (int m = 0; m < k; m++) {
                answer.add(tmp.toString());
            }
        }
        return answer.toArray(String[]::new);
    }
}